src/Support/FakeFacade.php

```diff
@@ -0,0 +1,198 @@
+<?php
+
+namespace Aloha\Twilio\Support;
+
+use Twilio\Exceptions\ConfigurationException;
+use Twilio\Exceptions\TwilioException;
+use Twilio\Rest\Api;
+use Twilio\Rest\Api\V2010;
+use Twilio\Rest\Api\V2010\Account\CallInstance;
+use Twilio\Rest\Api\V2010\Account\MessageInstance;
+use Twilio\Rest\Client;
+use Twilio\TwiML\TwiML;
+use Twilio\TwiML\VoiceResponse;
+use Aloha\Twilio\TwilioInterface;
+use Aloha\Twilio\Manager as Twilio;
+use Illuminate\Support\Arr;
+use PHPUnit\Framework\Assert;
```

Comment:
This will make phpunit a hard requirement (as opposed to dev-requirement) for this package. Not a good idea. Can we turn it around? for example:

```
getMessageSent(): Message
getCallSent(): Call
```

and let the test-suite which is testing it do the assertions.